Semgrep_output_v0.atd

```diff
@@ -576,6 +576,7 @@ type finding = {
   index: int;
   commit_date: string;
   syntactic_id: string;
+  pattern_based_id: string;
```

Comment:
I usually add a comment such as (* since semgrep 0.98 *)
and I usually make the field optional, since it's not present in older version, and sometimes we want
to remain backward compatible (especially in the code that reads the findings, such as the App,
which may get findings from older version of semgrep still).